### PR TITLE
Changed Colors to Light Blue theme

### LINKED ISSUE #151: Change color scheme to match that of webchat
**Actual Behaviour**

Currently, the colors are not similar to web chat app.

**Expected Behaviour**
Make the color to match, that of web chat.

### PR DESCRIPTION
**Pull request by dynamitechetan**
Fixes #151 
image

### CODE REVIEW

File: src/Utils/colors.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 let colors = {
     chatbox: '#4285f4',
-    header: "#546E7A",
-    sidebar:'#263238',
+    header: "#4285f4",
+    sidebar:'#0A518B',
     sidebarElements: "#212121",
```
Comment (saurabhjn76) [inline src/Utils/colors.js]: Please also change the sidebar Element color as per color scheme of web chat app.

**Comment by mariobehling** [comment]
@dynamitechetan As this project is now moving to the next stage PRs should have a test site. Please provide a test link with every PR.

File: src/Utils/colors.js (hunk 1 above)
Comment (dynamitechetan) [inline src/Utils/colors.js]: @saurabhjn76  header color is #4285f4 which is same as the color on chat.susi.ai 
They do not have a sidebar so I chose a darker blue color for the sidebar.
If you any suggestions for sidebar color please tell me.

**Comment by dynamitechetan** [comment]
@mariobehling  Yes, recently uday sent a fix for surge deployment. After that PR merges we can have deployment links for all future PRs.

File: src/Utils/colors.js (hunk 1 above)
Comment (saurabhjn76) [inline src/Utils/colors.js]: @dynamitechetan, In that case, it's fine. @uday96 if you have any suggestions please provide.

**Comment by mariobehling** [comment]
This has already partly been resolved. Ensure colors are also following the webchat on the login, signup etc.

Please provide a test site with each PR.

**Comment by dynamitechetan** [comment]
@mariobehling I am closing this one as Uday already made these changes in his PR.

**Pull request closed without merging**